services/web/server/src/simcore_service_webserver/config/server-docker-prod.yaml

```diff
@@ -56,4 +56,9 @@ projects:
 session:
   # python3 -c "from cryptography.fernet import Fernet; print(Fernet.generate_key())"
   secret_key: ${WEBSERVER_SESSION_SECRET_KEY}
+activity:
+  enabled: True
+  prometheus_host: http://prometheus
+  prometheus_port: 9090
+  prometheus_api_version: v1
 ...
```

Comment:
Here there should be environ variables so it's possible to configure from outside (see the other entries in the file)
e.g. WEBSERVER_PROMETHEUS_HOST, WEBSERVER_PROMETHEUS_PORT, WEBSERVER_PROMETHEUS_API_VERSION
they can have default values in the env file.